tools/src/linear-create-audit-issues/Configuration.ts

```diff
@@ -0,0 +1,199 @@
+        /* If this returns null then the issue should not be created nor updated */
+        description: string | null | ((moduleName: string) => Promise<string | null>);
+        labelNames: string[];
+      }[];
+    };
+  };
+} = {
+  teamName: 'Engineering',
+  projectName: 'Audit Modules',
+  labelName: 'Module',
+  umbrellaIssue: {
+    title: 'Audit Expo Modules Umbrella Issue',
+    description: `
+We haven't given many Expo modules the attention that they deserve.
+Most of the modules need some cleaning and improvements across many areas (docs, tests, API consistency, migrating native codebase, etc.).
+
+Below is the list of modules we're actively taking care of. Each of this module needs to be audited separately and ideally in every area mentioned.
+Beware that there are modules that might be deprecated or dropped completely in the near future.
+`,
+    labelNames: ['Module'],
+    childIssueTemplate: {
+      title: (moduleName: string) => `Audit ${moduleName}`,
+      description: `
+Go through the following sub-issues and ensure the module is taken care of in each aspect mentioned.
+Additionally add any missing sub-issues (reported on Twitter, Discord, etc.).
+If this module wraps some 3rd party API then, before working on it, ensure it won't to be deprecated in the nearest future in favour of the official alternative.
+`,
+      childIssueTemplates: [
+        {
+          title: (moduleName: string) => `Audit ${moduleName} - JS/TS API`,
+          description: `Look at the JS/TS API and ensure it's consistent and properly typed. Pay extra attention to our public API rules (like adding \`Async\` suffix to the method names)`,
+          labelNames: ['Module', 'API'],
+        },
+        {
+          title: (moduleName: string) => `Audit ${moduleName} - audit docs`,
+          description: `Ensure the docs for this module are up to date, comprehensive and easy to understand. Additionally ensure native english speaker has reviewed any changes.`,
+          labelNames: ['Module', 'Docs'],
+        },
+        {
+          title: (moduleName: string) => `Audit ${moduleName} - Web`,
+          description: `Ensure this module is properly implemented on the Web platform.`,
+          labelNames: ['Module', 'Web'],
+        },
+        {
+          title: (moduleName: string) => `Audit ${moduleName} - Android: rewrite to Kotlin`,
+          description: `Convert this module Android code to Kotlin.`,
+          labelNames: ['Module', 'Android'],
+        },
+        {
+          title: (moduleName: string) => `Audit ${moduleName} - Android: rewrite to SweetAPI`,
+          description: `Convert this module Android code to be SweetAPI-based`,
+          labelNames: ['Module', 'Android'],
+        },
+        {
+          title: (moduleName: string) => `Audit ${moduleName} - iOS: rewrite to Swift`,
+          description: `Convert this module iOS code to Swift.`,
+          labelNames: ['Module', 'iOS'],
+        },
```

Comment:
This task makes no sense — why would we rewrite to Swift and not to Sweet API? These two are actually the same.
```suggestion
```